src/test/java/io/github/mfvanek/pg/support/PostgresSqlClusterWrapper.java

```diff
@@ -63,6 +67,12 @@ final class PostgresSqlClusterWrapper {
 
         this.dataSourceForPrimary = PostgreSqlDataSourceHelper.buildDataSource(containerForPrimary);
         this.dataSourceForStandBy = PostgreSqlDataSourceHelper.buildDataSource(containerForStandBy);
+
+        Awaitility.await("Waiting for standby starts replicating data from primary")
```

Comment:
Also this could be done without awaitility by introducing new LogMessageWaitStrategy for standBy container creation.

Reply:
That works! Thanks